pkg/tinycrypt/Makefile.dep

```diff
@@ -0,0 +1,2 @@
+# Cifra is only supported by 32 bit architectures
```

Comment:
Copy/past mistake?

```suggestion
# TinyCrypt is only supported by 32 bit architectures
```

Reply:
Yep. But let's just drop the comment. The line below is pretty self-explaining